server/controllers/apply/applicationsController.ts

```diff
@@ -85,8 +86,17 @@ export default class ApplicationsController {
       } else {
         const offences = await this.personService.getOffences(req.user.token, crn)
         const indexOffence = offences.find(o => o.offenceId === offenceId)
-
-        const application = await this.applicationService.createApplication(req.user.token, crn, indexOffence)
+        let application: Application
+
+        try {
+          application = await this.applicationService.createApplication(req.user.token, crn, indexOffence)
+        } catch (error) {
+          if (error.status === 403) {
+            application = error.data
```

Comment:
Why do we want to continue here? I'd rather we just had some rough copy to start with instead of this potentially getting forgotten about and causing confusion further along in the journey

Reply:
I'm not clear on what should happen: 
- redirecting to CRN entry and showing an error? 
- allowing them to continue and showing some messaging?
- something else?

Reply:
Sorry, that was bad messaging on my part. We should show an error in the same way as if the CRN has not been found. Something like `The CRN ABC123 is not in your team's caseload`

Reply:
No probs - I should have checked earlier. 
I'll have a crack at that and add an integration test